README.md

```diff
@@ -1 +1,23 @@
-# graphql-authz
\ No newline at end of file
+# graphql-authz
+
+[![Coverage Status](https://coveralls.io/repos/github/casbin/graphql-authz/badge.svg?branch=master)](https://coveralls.io/github/casbin/graphql-authz?branch=master)[![Go](https://github.com/casbin/graphql-authz/actions/workflows/ci.yml/badge.svg)](https://github.com/casbin/graphql-authz/actions/workflows/ci.yml)[![Release](https://img.shields.io/github/release/casbin/graphql-authz.svg)](https://github.com/casbin/graphql-authz/releases/latest)[![Go Report Card](https://goreportcard.com/badge/github.com/casbin/graphql-authz)](https://goreportcard.com/report/github.com/casbin/graphql-authz)
+
+graphql-authz is a casbin binding of graphql, something like restful api. There're actions, like `enforce`, `getPolicies`, `addPolicy`, `removePolicy`, `updatePolicy`.
+
+## Install
+
+```bash
+go get -u github.com/casbin/graphql-authz
+```
+
+## Usage
+
+Enforce Example:
+
+```go
+e, _ := casbin.NewEnforcer("./examples/model.conf", "./examples/policy.csv")
+schema := authz.InitType(e)
+result := authz.Execute(`{enforce(sub:"alice" obj:"data1" act:"read"){sub obj act ok}}`, schema)
```

Comment:
How to let this repo act as a middleware of graphql? Does graphql provide a middleware mechanism?

Reply:
If we need to make a authz middleware, we could just using `Enforce` method, why graphql is needed?
Graphql is something like restful api, I don't think it should be used as middleware.
Maybe a http service will be more suitable, like https://github.com/closetool/graphql-authz/blob/51cffa025cfe733a4fd2ae6f1e512425c9dfccff/authz_test.go#L46